File: 2018/solutions/prob01.java
import java.io.*;

public class prob01 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader
                = new BufferedReader(new InputStreamReader(System.in));

        String line = null;
        while ((line = reader.readLine()) != null) {
            String[] data = line.split(" ");
            double liters = Double.parseDouble(data[0]);
            double density_kg_l = Double.parseDouble(data[1]);
            double mass_kg = (liters * density_kg_l);
            double mass_lb = mass_kg * 2.2;
            System.out.printf("%.2f\n", mass_lb);
        }
    }
}

File: 2018/solutions/prob01.c
#include <stdio.h>

int main(int argc, char **argv)
{
    double liters, density;
    double mass_kg, mass_lb;

    while (scanf("%lf %lf", &liters, &density) == 2) {
        mass_kg = liters * density;
        mass_lb = mass_kg * 2.2;
        printf("%.2f\n", mass_lb);
    }
    return 0;
}
